apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js

```diff
@@ -1,7 +1,8 @@
 window.configData = {
-	env_id: 'development',
+	env_id: 'production',
```

Comment:
This part is confusing, because it seems this file is used both in dev and production? But with the new checks, I think it should default to `production` so that the end bundle has the correct env. And if needed, one can change it to `development` in their local version.